cfme/intelligence/reports/reports.py

```diff
@@ -605,6 +631,28 @@ def step(self):
         self.prerequisite_view.configuration.item_select("Edit this Report")
 
 
+@navigator.register(Report, "Copy")
+class ReportCopy(CFMENavigateStep):
+    VIEW = ReportCopyView
+    prerequisite = NavigateToSibling("Details")
+
+    def step(self):
+        self.prerequisite_view.configuration.item_select("Copy this Report")
+
+
+@navigator.register(Report, "Timeline")
+class ReportTimeline(CFMENavigateStep):
+    VIEW = CloudIntelTimelinesView
+    prerequisite = NavigateToAttribute("appliance.server", "CloudIntelTimelines")
+
+    def step(self):
+        self.prerequisite_view.timelines.tree.click_path(
```

Comment:
It looks like `CloudIntelTimeslinesView.is_displayed` won't actually match this destination accurately.

I think you need a subclass for the view here, with an additional check against the accordion selection.